e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-minimal.spec.js

```diff
@@ -0,0 +1,46 @@
+import partials from '../../../../../../../partials';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import { assertMinimalExportContractSummaryListRows } from '../../../../../../../shared-test-assertions';
+
+const {
+  ROOT: INSURANCE_ROOT,
+  CHECK_YOUR_ANSWERS: { EXPORT_CONTRACT },
+} = INSURANCE_ROUTES;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswers;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Check your answers - Export contract - As an exporter, I want to be able to review my input for the export contract again, So that I can do a final review of the information I previously input before submitting my application', () => {
```

Comment:
Should we add summary list to the context?

Reply:
Good shout, added in.